doc/source/_res/useful-links.txt

```diff
@@ -0,0 +1,12 @@
+Useful Links
+=======================
+
+The following are useful links:
```

Comment:
I would remove this sentence since it duplicates the heading.

Reply:
Done.